style.css

```diff
@@ -0,0 +1,642 @@
+body {
+  overflow-x: hidden;
+  background-color: #c9bdbd;
+  font-family: Arial, sans-serif;
+}
+
+nav {
+  display: flex;
+  width: 100%;
+  justify-content: space-around;
+  background-color: #363232;
+  align-items: center;
+  text-align: center;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: bold;
+}
+
+nav div {
+  padding: 25px;
+}
+
+.colored {
+  background-color: #1f1d1d;
+  border-radius: 5px;
+}
+
+.nav-links {
+  flex-direction: row;
+  display: flex;
+  justify-content: space-between;
+  width: 70%;
+}
+
+.nav-logo {
+  width: 10%;
+}
+
+.search-bar {
+  width: 20%;
+  background-color: white;
+  border-color: transparent;
+  border-radius: 5px;
+  color: gray;
+  font-size: 20px;
+  margin-right: 50px;
+}
+
+.nav-links a,
+.nav-logo a {
+  padding: 20px;
+  color: white;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+.main-one-top a {
+  font-size: 25px;
+  font-weight: bold;
+  padding: 15px;
+  background-color: white;
+  margin: 0;
+  width: max-content;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+}
+
+.main-one-right a {
+  font-size: 30px;
+  padding: 15px;
+  background-color: white;
+  margin: 0;
+  width: max-content;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+}
+
+.main-one-grid {
+  background-color: #302d2d;
+  grid-column: 1/span3;
+  grid-row: 1/span4;
+}
+
+.main-one-grid div {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 200px 50px;
+}
+
+.main-one-grid a {
+  color: white;
+  font-size: 45px;
+  font-weight: bold;
+  text-decoration: none;
+  padding: 30px 0;
+}
+
+.main-one-grid1 {
+  grid-column: 4/span3;
+  grid-row: 1/span4;
+  background-color: #363030;
+}
+
+.main-one-grid1 div {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 200px 50px;
+}
+
+.main-one-grid1 a {
+  color: white;
+  font-size: 45px;
+  font-weight: bold;
+  text-decoration: none;
+  padding: 30px 0;
+}
+
+.main-one-grid2 {
+  grid-column: 1/span3;
+  grid-row: 5/span3;
+  background-color: #292626;
+  padding: 120px 50px;
+}
+
+.main-one-grid2 div {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.main-one-grid2 a {
+  color: white;
+  font-size: 45px;
+  font-weight: bold;
+  text-decoration: none;
+  padding: 30px 0;
+}
+
+.main-one-grid3 {
+  grid-column: 4/span3;
+  grid-row: 5/span3;
+  background-color: #302d2d;
+  padding: 120px 50px;
+}
+
+.main-one-grid3 div {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.main-one-grid3 a {
+  color: white;
+  font-size: 45px;
+  font-weight: bold;
+  text-decoration: none;
+  padding: 30px 0;
+}
+
+.text-div a {
+  color: #1f1d1d;
+  text-decoration: none;
+}
+
+.box-div {
+  background-color: white;
+  width: 100%;
+  height: 100%;
+  border-radius: 10px;
```

Comment:
- [ ] `border-radius: 5px`, `border-radius: 10px` is ussed repeated times in the file, it would be better to create something like the next selectors and apply the selector as a class in each element that uses that kind of border radious. This will help to maintain the code DRY.
```css
br-5 {
   border-radius: 5px;
}

br-10 {
  border-radius: 10px;
}
```

Reply:
Thanks for the suggestions, made the changes accordingly.